utils.js

```diff
@@ -16,6 +18,16 @@ module.exports = {
         } else {
             res.send(401);
         }
+    },
+
+    /* Hash provided token */
+    hashToken: async function (token) {
```

Comment:
Is this automatically used by express? I don't see any other references to `hashToken` in this PR.

Reply:
Its being used [here](https://github.com/openmrs/openmrs-contrib-atlas/pull/89/files#diff-1b10389ce019116e9ba8118b5f2719ecR316).